exasol_integration_test_docker_environment/lib/api/build_test_container.py

```diff
@@ -0,0 +1,50 @@
+from typing import Tuple, Optional
+
+from exasol_integration_test_docker_environment.cli.common import set_build_config, \
+    set_docker_repository_config, run_task, generate_root_task
+from exasol_integration_test_docker_environment.lib.test_environment.analyze_test_container import AnalyzeTestContainer, \
+    DockerTestContainerBuild
+
+
+def build_test_container(
+        force_rebuild: bool = False,
+        force_rebuild_from: Tuple[str, ...] = tuple(),
+        force_pull: bool = False,
+        output_directory: str = ".build_output",
+        temporary_base_directory: str = "/tmp",
+        log_build_context_content: bool = False,
+        cache_directory: Optional[str] = None,
+        build_name: Optional[str] = None,
+        source_docker_repository_name: str = 'exasol/script-language-container',
```

Comment:
We should create a new docker repo and change this default in another PR. Could you create a ticket, please.

Reply:
Also, we should create a constant for the default value and use everywhere, we probably have the string 8 times or more standing directly in the code

Reply:
Done (created #238) and done